roles/interfaces/tasks/el8.yml

```diff
@@ -130,9 +101,53 @@
     daemon_reload: true
   become: true
 
+- name: Find old network interface configs in ifcfg script format.
+  ansible.builtin.find:
+    paths: /etc/sysconfig/network-scripts/
+    patterns: 'ifcfg-.*'
+    use_regex: true
+  register: interfaces_found_ifcfg_scripts
+
+- name: Migrate network interface configs from ifcfg script to keyfile format.
```

Comment:
here is no failed when. What happens when migration is not successful? Does it get still deleted in the Line 140 or does it fail here?
It should be deleted *only* when successfully migrated, right?